### PR TITLE
feat: add exclusive post type

### PR DESCRIPTION
**Pull request by sk-yagi**

### CODE REVIEW

File: modules/business-types/index.ts [hunk 1]
```diff
@@ -94,6 +96,11 @@ export type ProjectAnnouncement = {
   censorship?: string[];
 };
 
+export type ExclusivePost = Omit<ProjectAnnouncement, "body"> & {
+  body: CipherMeta & { data: Base64 };
+  totalActiveStakeToView: LovelaceAmount;
```
Comment (sk-shishi) [inline modules/business-types/index.ts:101]:
```ts
exclusive: { tier: number }
```

File: modules/business-types/index.ts [hunk 2]
```diff
@@ -94,6 +96,11 @@ export type ProjectAnnouncement = {
   censorship?: string[];
 };
 
+export type ExclusivePost = Omit<ProjectAnnouncement, "body"> & {
+  body: CipherMeta & { data: Base64 };
```
Comment (sk-shishi) [inline modules/business-types/index.ts:100]: Introduce an `CipherText<_>` type in the `crypt` module.

```ts
export CipherText<_> = Base64;
```

Comment (sk-shishi) [reply modules/business-types/index.ts:100]: Then this should be
```ts
body: CipherMeta & { ciphertext: CipherText<JSONContent> }
```

**Comment by sk-yagi** [comment]
This will be moved to an exclusive branch

**Pull request closed without merging**